Makefile

```diff
@@ -13,11 +13,12 @@ GOBIN_DEFAULT := $(GOPATH)/bin
 export GOBIN ?= $(GOBIN_DEFAULT)
 TESTARGS_DEFAULT := "-v"
 export TESTARGS ?= $(TESTARGS_DEFAULT)
-PKG := $(shell awk  -F "\"" '/^ignored = / { print $$2 }' Gopkg.toml)
 DEST := $(GOPATH)/src/$(GIT_HOST)/$(BASE_DIR)
 SOURCES := $(shell find $(DEST) -name '*.go')
 
-HAS_DEP := $(shell command -v dep;)
+# Kctive module mode, as we use go modules to manage dependencies
```

Comment:
What is `Kctive`

Reply:
This is a typo,  should be `# Active module mode ...`